spec/jobs/education_form/create_daily_year_to_date_report_spec.rb

```diff
@@ -11,6 +11,10 @@ def get_education_form_fixture(filename)
     described_class.new
   end
 
+  before do
+    allow_any_instance_of(EducationBenefitsClaim).to receive(:create_education_benefits_submission)
```

Comment:
are you not concerned with what it needs to return? If not, consider changing this to an expectation. Usually allow is used to stub something when you need it to return a special type and expect is used when you need to test that it's called.

Reply:
i don't want it to be called at all, it's tested in another spec. before i added that line whenever i would call `create(:education_benefits_submission)` it would create 2 submissions because education_benefits_submission invokes the education_benefits_claim factory and that invokes the `:create_education_benefits_submission` callback, it makes creating test data too complicated.

Reply:
I think you could also do: 

`EducationBenefitsClaim.skip_callback(:create_education_benefits_submission)` in RSpec.